src/DynamoPackages/PackageLoader.cs

```diff
@@ -278,11 +279,27 @@ public void LoadAll(LoadPackageParams loadPackageParams)
         public void LoadPackages(IEnumerable<Package> packages)
         {
             var enumerable = packages.ToList();
+
+            // This fix is in reference to the crash reported in task: https://jira.autodesk.com/browse/DYN-2101
+            // TODO: we will be re-evaluating this workflow, to find the best clean solution. 
```

Comment:
can you describe the refactor problem and file it as a jira task linking the task number here?

Reply:
Added a TODO task in JIRA: https://jira.autodesk.com/browse/DYN-2120

Reply:
can you add this DYN task number in the comment?